src/components/PeerNameDisplay/usePeerNameDisplay.ts

```diff
@@ -43,10 +43,16 @@ export const usePeerNameDisplay = () => {
     return displayUsername
   }
 
+  // Hàm mới để rút gọn ID người dùng
+  const getShortenedUserId = (userId: string) => {
+    return `${userId.slice(0, 6)}...${userId.slice(-3)}`
```

Comment:
Let's gracefully handle unexpectedly short user IDs here. For instance, if the user ID is 12 characters or fewer, just return `userId`.